jelatyna-backend/src/main/java/pl/confitura/jelatyna/infrastructure/security/TokenService.java

```diff
@@ -30,7 +30,7 @@ public String asToken(User user) {
                 .setClaims(new HashMap<String, Object>() {{
                     put("isAdmin", user.isAdmin());
                     put("isVolunteer", user.isVolunteer());
-                    put("isSpeaker", user.isSpeaker());
+                    put("isSpeaker", user.getSpeaker());
```

Comment:
`getSpeaker` ? to to zwraca obiekt speakera?

Reply:
dobra uwaga, dzieki